Sources/main.cpp

```diff
@@ -99,6 +101,12 @@ static int PlatformMain(std::string const &host, std::string const &port) {
 
   std::unique_ptr<Socket> serverSocket = CreateSocket(host, port, false);
 
+#if defined(OS_POSIX)
```

Comment:
I don't think we should use ifdef's in cases where we don't actually need them. Presumably it's possible to implement a daemon on non-POSIX systems, and we just haven't gotten around to it. Using an ifdef makes it look to future developers like it's not a possible future option, whereas a dummy failing function indicates that it's a possible new feature that we haven't yet implemented.